### DISCUSSION TITLE
Discussion labels wrap weirdly on mobile

### DISCUSSION THREAD

**Discussion by ajsirota** [community/community#64137] [Mobile]
![image](https://github.com/community/community/assets/67526318/854b80df-74ad-4bbb-84af-438bacdbb870)
This doesn't seem to happen on issues and PRs, only discussions. 

<details>
<summary>ℹ️ Additional Debug Info</summary>
GitHub for iOS, Version 1.127.0 (107661963)
Device: iPhone 13 (iOS 16.6)
</details>

**Comment by GhostOf0days** [comment]
This is intended behavior and makes sense. How would you want it to wrap? I don't think centering the last row is a great idea for cards/labels.

**Reply by ajsirota** [reply]
I'm talking about how the text of one label is cut off (there are not separate labels `Product Fee` and `dback`, there's just `Product Feedback`)

**Reply by GhostOf0days** [reply]
Oh I didn't see that. You're right. That should go in the next row.

**Comment by rainbow1016** [comment]
very good question!

**Comment by YaowamalSomthai** [comment]
> ![image](https://user-images.githubusercontent.com/67526318/261461946-854b80df-74ad-4bbb-84af-438bacdbb870.jpeg) This doesn't seem to happen on issues and PRs, only discussions.
> 
> ℹ️ Additional Debug Info
> GitHub for iOS, Version 1.127.0 (107661963) Device: iPhone 13 (iOS 16.6)

**Comment by YaowamalSomthai** [comment]
https://github.com/orgs/community/discussions/64137#discussion-5532152
